R/dataProcessPlots.R

```diff
@@ -473,11 +583,12 @@ dataProcessPlots = function(
                                   text.size, text.angle, legend.size, 
                                   dot.size.condition, yaxis.name)
     print(con_plot)
+    plots <- c(plots, list(con_plot))
```

Comment:
let's improve this as we update the plotly code [again: leftover from old coding style, should be something like plots[[i]] = con_plot)

Reply:
Commit [better list handling](https://github.com/Vitek-Lab/MSstats/pull/107/commits/5c098d0c5d50d0300d8bca65672c0d6e9ec382e7)